src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -254,4 +243,33 @@ public JsonApiException toApiException() {
   public JsonApiException toApiException(Response.Status httpStatus) {
     return new JsonApiException(this, message, null, httpStatus);
   }
+
+  /**
+   * Helper class to cache loading of settings from the configuration. This is used to avoid having
+   * to access Config during Enum class initialization. It will also prevent repeated configuration
+   * loading calls.
+   */
+  static class ExtendError {
+    private static final ExtendError instance = new ExtendError();
```

Comment:
Instead of private static instance and accessor, could have exposed this as `public`. But looks to me slightly cleaner from caller this way.